aerleon/lib/openconfig.py

```diff
@@ -259,16 +270,32 @@ def _TranslatePolicy(self, pol: Policy, exp_info: int) -> None:
                     term_address_families = [address_family]
                 for term_af in term_address_families:
                     t = Term(term, term_af)
+
                     for rule in t.ConvertToDict():
                         total_rule_count += 1
                         rule['sequence-id'] = total_rule_count * 5
-                        self.oc_policies.append(rule)
+                        oc_acl_entries.append(rule)
+
+            if address_family == 'mixed':
```

Comment:
It would be cleaner to make a dict of these so all you ask for is 
```
family_map = {
   'mixed': 'ACL_MIXED',
   'inet6': 'ACL_IPV6',
   'inet': 'ACL_IPV4'
   }
oc_type = family_type[address_family]
```
The map should however be located near line 211 close to where `_PLATFORM` and `SUFFIX` are defined.

Reply:
Good idea!